decred/decred/dcr/account.py

```diff
@@ -138,6 +138,22 @@ def __init__(self, count=0, value=0):
         self.value = value
 
 
+def unblob_check(name, version, actual_pushes, expected_pushes):
```

Comment:
As versions change this function won't be as useful, but it works for now.

Reply:
I successfully fought the inner urge to make it more general than absolutely needed right now. :slightly_smiling_face: It'll be easy enough to adjust it when the need actually arises.

Reply:
So you knew it was going to change in #53. :stuck_out_tongue: